ironfish/src/mining/director.ts

```diff
@@ -410,8 +410,7 @@ export class MiningDirector {
 
     this.recentBlocks.set(this.miningRequestId, newBlock)
 
-    const canRetry = target.asBigInt() < Target.maxTarget().asBigInt()
-    return canRetry
+    return target.asBigInt() < Target.maxTarget().asBigInt()
```

Comment:
This code is less clear now. It's unclear what is being returned from this method.

Reply:
Hey @NullSoldier,

From my knowledge doing that is not a best practice. I could on the other hand add some comments to explain the what the return is.

Reply:
Added comments for better readability